scripts/generators/csv_generator.py

```diff
@@ -39,14 +42,15 @@ def save_csv(file, sorted_fields, version):
                 field_set = key_parts[0]
 
             indexed = str(field.get('index', True)).lower()
+            example = ecs_helpers.sort_field_if_dict_as_str(field.get('example', ''))
```

Comment:
force the examples to be sorted by key so it doesn't cause the generated files to change.